src/otx/recipe/detection/atss_r50_fpn.yaml

```diff
@@ -10,9 +10,6 @@ optimizer:
     weight_decay: 0.0
 
 scheduler:
-  - class_path: otx.algo.schedulers.warmup_schedulers.LinearWarmupScheduler
-    init_args:
-      num_warmup_steps: 3
```

Comment:
Is this model also completely devoid of warmup?

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/3035/commits/3ccd4b1946381915bc2c179da8746a23a82f9a56